R/load_forecasts.R

```diff
@@ -6,19 +6,21 @@
 #' @param forecast_dates The forecast date of forecasts to retrieve.
 #' Default to all valid forecast dates.
 #' The function will throw an error if all dates in this parameter are invalid forecast dates in Zoltar.
-#' @param locations list of fips. Default to all locations with available forecasts.
+#' @param locations list of location codes Default to all locations with available forecasts.
 #' @param types Character vector specifying type of forecasts to load: “quantile” 
 #' or “point”. Default to all valid forecast types.
 #' @param targets character vector of targets to retrieve, for example
 #' c('1 wk ahead cum death', '2 wk ahead cum death'). 
 #' Default to NULL which stands for all valid targets.
 #' @param source string specifying where forecasts will be loaded from: either 
 #' "local_hub_repo" or "zoltar". Default to "zoltar"
-#' @param hub_repo_path path to local clone of the reichlab/covid19-forecast-hub
+#' @param hub_repo_path path to local clone of the forecast hub
 #' repository
 #' @param as_of a date in YYYY-MM-DD format to load forecasts submitted as of this date. 
-#' Default to NULL to load the latest version. Only available when source is "zoltar" now. 
-#' @param verbose a boolean for printing messages on zoltar job status. Default to TRUE.
+#' Default to NULL to load the latest version.
+#' @param verbose a boolean for printing messages on zoltar job status poll
+#' @param hub character vector, where the first element indicates the hub
```

Comment:
Based on my understanding, the expected behavior of load_forecasts() and all other related function should be reading forecasts from a single hub, right? Could we just not set any default to this parameter and add `match.arg(hub, choices = c("US", "ECDC"), several.ok = FALSE)` to check the user input? If the user wants to read from both hubs, the user could do a map_dfr on results from this function. Does it make sense in this case?

Reply:
r.e. the default for the hub -- our thought was to make the US hub a default so that existing code doesn't need to be updated to add a hub argument.  if this is essentially suggesting a list of valid options and only the first would be used, I think that's fine (and standard practice in R, though confusing)

Reply:
in the code it only ever evaluates the first element of that vector - so the US is actually the default. The only reason I specified a vector as the default is that users would immediately see what they could choose from. Happy to move that to the documentation if you prefer